doc/maintaining/installing/install-from-package-es.rst

```diff
@@ -0,0 +1,206 @@
+.. include:: /\_substitutions.rst
```

Comment:
This should not be escaped:
```suggestion
.. include:: /_substitutions.rst
```

In general, none of the Sphynx directives should be touched at all